src/pages/index.js

```diff
@@ -90,10 +188,16 @@ function getCardElement(data) {
     cardLikeBtn.classList.toggle("card__like-button_liked");
   });
 
-  const deleteButton = cardElement.querySelector(".card__delete-button");
+  deleteButton.addEventListener("click", () =>
+    handleDeleteCard(cardElement, data._id)
+  );
+
+  deleteModalCloseBtn.addEventListener("click", () => {
+    closeModal(deleteModal);
+  });
 
-  deleteButton.addEventListener("click", () => {
-    cardElement.remove();
+  deleteModalCancelBtn.addEventListener("click", () => {
+    closeModal(deleteModal);
```

Comment:
`deleteModalCloseBtn` and `deleteModalCancelBtn` should be handled **only once outside** `getCardElement`. It's not about cards. It's about the confirmation `modal`